src/test/java/seedu/address/model/book/BookTest.java

```diff
@@ -15,10 +14,6 @@
 
 import org.junit.jupiter.api.Test;
 
-//import seedu.address.commons.util.DateUtil;
-//import seedu.address.model.borrower.BorrowerId;
-//import seedu.address.model.loan.Loan;
```

Comment:
Are these imports still needed?

Reply:
Can add them back when we need them, none of the tests using them at the moment